Automatic Validation ended with: 
> 2026-04-14T17:25:08.6965940Z ##[error] Manifest Error: Field usage requires verified publishers. [Icons]

(Automated response - build 1248.)

--- a/manifests/t/TRGamer-tech/FluentTaskScheduler/1.8.0/TRGamer-tech.FluentTaskScheduler.locale.en-US.yaml
+++ b/manifests/t/TRGamer-tech/FluentTaskScheduler/1.8.0/TRGamer-tech.FluentTaskScheduler.locale.en-US.yaml
@@ -28,11 +28,7 @@ Tags:
   - productivity
 Documentations:
   - DocumentLabel: README
-    DocumentUrl: https://github.com/TRGamer-tech/FluentTaskScheduler/blob/main/README.md
-Icons:
-  - IconUrl: https://raw.githubusercontent.com/TRGamer-tech/FluentTaskScheduler/main/Assets/AppIcon.png
-    IconFileType: png
-    IconTheme: default
+    DocumentUrl: https://github.com/TRGamer-tech/FluentTaskScheduler/blob/main/README.md
 ReleaseNotesUrl: https://github.com/TRGamer-tech/FluentTaskScheduler/releases/tag/V1.8.0
 ManifestType: defaultLocale
 ManifestVersion: 1.12.0

--- a/manifests/t/TRGamer-tech/FluentTaskScheduler/1.8.0/TRGamer-tech.FluentTaskScheduler.locale.en-US.yaml
+++ b/manifests/t/TRGamer-tech/FluentTaskScheduler/1.8.0/TRGamer-tech.FluentTaskScheduler.locale.en-US.yaml
@@ -28,7 +28,7 @@ Tags:
   - productivity
 Documentations:
   - DocumentLabel: README
-    DocumentUrl: https://github.com/TRGamer-tech/FluentTaskScheduler/blob/main/README.md
+    DocumentUrl: https://github.com/TRGamer-tech/FluentTaskScheduler/blob/main/README.md
 ReleaseNotesUrl: https://github.com/TRGamer-tech/FluentTaskScheduler/releases/tag/V1.8.0
 ManifestType: defaultLocale
 ManifestVersion: 1.12.0

